js/slider/shared/Range.svelte

```diff
@@ -41,13 +42,40 @@
 		dispatch("release", value);
 		value = Math.min(Math.max(value, minimum), maximum);
 	}
+
+	var rangeInputs = null;
+	var numberInputs = null;
```

Comment:
I think we don't need to define global variables now. I wrote this before because at that time gradio-app didn't have event listeners like "render", and we needed to detect if sliders are added to the DOM tree.But now it does, maybe just define & assignment in the internal function would be better.